src/components/MarkerMetadata/MarkerMetadata.js

```diff
@@ -52,21 +54,21 @@ function DisplayMarker(props) {
         </Grid>
       </Grid>
 
-      <Typography variant="subtitle1">
-        {summary || (
-          <span style={{ color: '#999' }}>
-            This annotation has no summary, click to add one
-          </span>
-        )}
-      </Typography>
+      { summary &&
+         <Typography variant="subtitle1">
+            <span style={{ color: '#999' }}>
+              This annotation has no summary, click to add one
```

Comment:
This may be a bug, since the summary isn't displayed.

```suggestion
              { summary }
```

Reply:
:+1: Good catch.